code/extensions/FormSpamProtectionExtension.php

```diff
@@ -74,6 +74,8 @@ public function enableSpamProtection($options = array())
         // captcha form field name (must be unique)
         if (isset($options['name'])) {
             $name = $options['name'];
+        } else if($field_name = Config::inst()->get('FormSpamProtectionExtension', 'field_name')) {
```

Comment:
Can you please add the `private static $field_name = 'Captcha';` to this class, then you can have one fewer condition. :)